node/impl/full/sync.go

```diff
@@ -51,6 +51,17 @@ func (a *SyncAPI) SyncState(ctx context.Context) (*api.SyncState, error) {
 }
 
 func (a *SyncAPI) SyncSubmitBlock(ctx context.Context, blk *types.BlockMsg) error {
+	err := a.SyncBlock(ctx, blk)
+	b, err := blk.Serialize()
+	if err != nil {
+		return xerrors.Errorf("serializing block for pubsub publishing failed: %w", err)
```

Comment:
xerrors is deprecated. Let's replace it with `errors`/fmt.Errorf as we discussed here and below